File: uri/bonus/bonus.ml

let meusalario salario montante = 
    salario +. ((montante *. 15.0) /.100.0);;

let nome = read_line();;    
let salario = float_of_string ( read_line() );;
let montante = float_of_string( read_line() );;

let resultado = meusalario salario montante;;
Printf.printf "TOTAL = R$ %.2f\n" ( resultado ) ;;

File: uri/bonus/bonus.lua
local salario, montante

function meusalario( salario, montante )
    return (salario + ((montante * 15)/100))
end

nome = io.read()
salario = io.read("*n")
montante = io.read("*n")

resultado = string.format("%.2f", meusalario(salario, montante))
print ("TOTAL = R$ "..resultado)